francis/__main__.py

```diff
@@ -209,21 +209,30 @@ def listen(audio_sample, model_path):
     pre_df = io.load_file_into_df(audio_sample)
 
     # preprocess
+    preprocess_bar = default_bar(
+        "noise reducing and high pass filtering \t\t\t", len(pre_df)
+    )
     the_df = preprocess.process(
-        pre_df, CONFIG["SAMPLE_RATE"], CONFIG["PREPROCESSING_ON"]
+        pre_df,
+        CONFIG["SAMPLE_RATE"],
+        CONFIG["PREPROCESSING_ON"],
+        bar_config=preprocess_bar,
     )
```

Comment:
Thanks for doing it this way, Its not the best way of doing it, but its the clearest way, while keeping seperation of logic. The downside is that it creates a little bit of clutter!